tests/datasets/test_synthetic_lm_data.py

```diff
@@ -20,7 +20,11 @@ def generate_parameter_configs(keys, num_replicas=1):
 
     config_combinations = []
     for combo in product(*[config_options[i] for i in keys]):
-        config_combinations.append([dict(zip(keys, combo)) for _ in range(num_replicas)])
+        config = dict(zip(keys, combo))
+        if "tokenizer_family" in keys:
```

Comment:
When would this not be true?

Reply:
you mean when do we not pass `tokenizer_family` in the keys argument? It's sometimes not passed in one of the tests.  Not sure why but it's here https://github.com/mosaicml/composer/blob/fb4beb134ac9f2b88cb562e1ad06e19686092289/tests/datasets/test_synthetic_lm_data.py#L46

Reply:
resolved per @moinnadeem